deploy/playbooks/deploy.yml

```diff
@@ -16,3 +16,4 @@
      repos_root: "/opt/repos"
      branch: "master"
      development_server: false
+     app_user: "nginx"
```

Comment:
+1 to a "chacra" user. We'd just need to write the ansible to ensure the user exists as well.